lab_1_classify_by_unigrams/main.py

```diff
@@ -4,104 +4,71 @@
 """
 
 
-def tokenize(text: str) -> list[str] | None:
-    """
-    Splits a text into tokens, converts the tokens into lowercase,
-    removes punctuation, digits and other symbols
-    :param text: a text
-    :return: a list of lower-cased tokens without punctuation
-    """
+def tokenize(text):
+    text = text.lower()
+    for i in text:
+        if not i.isalnum() and i != ' ':
+            text = text.replace(i, '')
+    for i in text:
+        if i == ' ':
```

Comment:
is it possible to encounter `' '` when they are all deleted previously? 

Also, it is not a good practice to modify the sequence that you're modifying.

Reply:
I don't know why, but '  ' remain in the token list if I dont replace ' ',''

Reply:
Consider filtrating the text instead of altering it.

Reply:
Done